sdv/demo.py

```diff
@@ -333,7 +333,9 @@ def _load_tabular_dummy():
         'address': adresses,
         'age': age,
         'age_when_joined': age_when_joined,
-        'years_in_the_company': age - age_when_joined
+        'years_in_the_company': age - age_when_joined,
+        'joined_in_first_semester': [1.0, 0.0, 1.0, 1.0, 0.0, 0.0] * 2,
```

Comment:
minor: the demo table is called `employees` right? I usually don't think of employees as having joined in semesters as that's usually a school related concept. Can we name it something else?

Reply:
Honestly, I just had trouble coming up with a better idea :disappointed:, since I can only add two one hot columns to the demo (don't want to clutter it). Do you have any suggestions?

Reply:
Maybe whether or not the employee was hired as a new grad? Or whether or not they were referred by another employee?